jekyll/_cci2/browser-testing.md

```diff
@@ -239,19 +225,22 @@ $ ssh -p PORT ubuntu@IP_ADDRESS -L 5900:localhost:5900
 
 CircleCI also supports X11 forwarding over SSH. X11 forwarding is similar to VNC &mdash; you can interact with the browser running on CircleCI from your local machine.
 
-1. Install an X Window System on your computer. If you're using macOS, consider [XQuartz](http://xquartz.macosforge.org/landing/).
+1. Install an X Window System on your computer. If you're using macOS, consider [XQuartz (http://xquartz.macosforge.org/landing/).
```

Comment:
probably didn't mean to remove this closing bracket?

Reply:
Fixed